db/l2db/l2db.go

```diff
@@ -375,14 +375,12 @@ func (l2db *L2DB) Purge(currentBatchNum common.BatchNum) (err error) {
 		`DELETE FROM tx_pool WHERE (
 			batch_num < $1 AND (state = $2 OR state = $3)
 		) OR (
-			(SELECT count(*) FROM tx_pool WHERE state = $4) > $5 
-			AND timestamp < $6 AND state = $4
+			state = $4 AND timestamp < $5
```

Comment:
As mentioned previously in a meeting, it would be good to update the config documentation regarding this change: 
https://github.com/hermeznetwork/hermez-node/blob/c25d8ca95c5e6cd35d66b2da35df18d76de410e1/config/config.go#L152